tests/fixtures/typescript/basics/interface-with-all-property-types.src.ts

```diff
@@ -0,0 +1,15 @@
+interface Foo {
```

Comment:
I can split them into two test. Not sure which ones are invalid exactly but I think the computed property don't make sense in interfaces.